.github/workflows/documentation.yml

```diff
@@ -0,0 +1,47 @@
+name: Documentation
+
+on:
+  push:
+    branches: [master, litepcie-update]
```

Comment:
default branch names have changed:
develop - for the latest code
stable - for releases